src/pages/findYourBallot.module.scss

```diff
@@ -1,9 +1,8 @@
-.body {
-  padding: 4.5rem 5rem;
+.container {
+  padding: 10rem 20rem;
```

Comment:
that's a lot of padding! i guess this is the one with the tiny iframe?

Reply:
Diane and Irina mentioned they wanted it to look more like the About Us page, so I brought the padding to be in line with it.